CliClient/locales/de_DE.po

```diff
@@ -0,0 +1,957 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR Laurent Cozic
+# This file is distributed under the same license as the Joplin-CLI package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Joplin-CLI 1.0.0\n"
+"Report-Msgid-Bugs-To: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: \n"
+"X-Generator: Poedit 2.0.4\n"
+"Last-Translator: \n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: de_DE\n"
+
+msgid "Give focus to next pane"
+msgstr "Fokussiere das nächste Fenster"
+
+msgid "Give focus to previous pane"
+msgstr "Fokussiere das vorherige Fenster"
+
+msgid "Enter command line mode"
+msgstr "Wechsle zum Terminal-Modus"
+
+msgid "Exit command line mode"
+msgstr "Verlasse den Terminal-Modus"
+
+msgid "Edit the selected note"
+msgstr "Bearbeite die ausgewählte Notiz"
+
+msgid "Cancel the current command."
+msgstr "Breche den momentanen Befehl ab."
+
+msgid "Exit the application."
+msgstr "Verlasse das Programm."
+
+msgid "Delete the currently selected note or notebook."
+msgstr ""
+"Lösche die momentan ausgewählte Notiz oder das momentan ausgewählte "
+"Notizbuch."
+
+msgid "To delete a tag, untag the associated notes."
+msgstr "Um eine Markierung zu löschen, entmarkiere markierte Notizen."
+
+msgid "Please select the note or notebook to be deleted first."
+msgstr ""
+"Bitte wählen Sie zuert eine Notiz oder ein Notizbuch aus, das gelöscht "
+"werden soll."
+
+msgid "Set a to-do as completed / not completed"
+msgstr "Markiere ein ToDo as abgeschlossen / nicht abgeschlossen"
```

Comment:
Probably rather `To-Do`,  I think: https://dict.leo.org/forum/viewUnsolvedquery.php?idThread=538311&idForum=2&lang=en&lp=ende

Reply:
Yeah, To-Do sounds better, thanks :)